CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # Changelog
 
+## [1.20](https://github.com/brave/brave-ios/releases/tag/v1.20)
+
+ - Implemented capability to set Brave as default browser. ([#2785](https://github.com/brave/brave-ios/issues/2785))
```

Comment:
I think using `Added ability to set Brave as the default browser` might sound a bit better/easier to understand. I don't think there's a need to use the word `capability` as that's a given due as it's being implemented/added.